src/renderer/component/transactionList/view.jsx

```diff
@@ -111,31 +104,14 @@ class TransactionList extends React.PureComponent<Props, State> {
                 />
               }
             >
-              <option value="all">{__('All')}</option>
-              <option value={TRANSACTIONS.SPEND}>
-                {__(`${this.capitalize(TRANSACTIONS.SPEND)}s`)}
-              </option>
-              <option value={TRANSACTIONS.RECEIVE}>
-                {__(`${this.capitalize(TRANSACTIONS.RECEIVE)}s`)}
-              </option>
-              <option value={TRANSACTIONS.PUBLISH}>
-                {__(`${this.capitalize(TRANSACTIONS.PUBLISH)}es`)}
-              </option>
-              <option value={TRANSACTIONS.CHANNEL}>
-                {__(`${this.capitalize(TRANSACTIONS.CHANNEL)}s`)}
-              </option>
-              <option value={TRANSACTIONS.TIP}>
-                {__(`${this.capitalize(TRANSACTIONS.TIP)}s`)}
-              </option>
-              <option value={TRANSACTIONS.SUPPORT}>
-                {__(`${this.capitalize(TRANSACTIONS.SUPPORT)}s`)}
-              </option>
-              <option value={TRANSACTIONS.UPDATE}>
-                {__(`${this.capitalize(TRANSACTIONS.UPDATE)}s`)}
-              </option>
-              <option value={TRANSACTIONS.ABANDON}>
-                {__(`${this.capitalize(TRANSACTIONS.ABANDON)}s`)}
-              </option>
+              {transactionTypes
+                .concat('all')
```

Comment:
After looking at this now, I think we should just add a new `TRANSACTION` value for `"all"` instead of referencing it in a couple places. Sorry for merging the lbry-redux PR before realizing this.